js/main.js

```diff
@@ -94,10 +131,38 @@ function viewSwap(viewName) {
   }
 }
 
+// shows Entries page when the Entries tab is clicked
 $nav.addEventListener('click', function () {
   viewSwap('entries');
 });
 
+// shows the Entry Form page when NEW button is clicked
 $newButton.addEventListener('click', function () {
   viewSwap('entry-form');
 });
+
+// switch to Edit Entries page when pen icon is clicked
+$ul.addEventListener('click', function (event) {
+  viewSwap('entry-form');
```

Comment:
I would call this at the end of the function, so you swap after you fill in all the inputs